opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
One other thing: I don't like how dapper+zipkin represent an RPC as a single span... it seems like a better fit (IMO) to model it as two closely-related spans. What do you think about that? Too painful of a transition? If so, though, I could imagine annotations here that are specialized for the source and target roles specifically... perhaps their `Add()` methods would take different sets of parameters.